app/code/core/Mage/Review/etc/adminhtml.xml

```diff
@@ -31,23 +31,23 @@
                             <children>
                                 <pending translate="title" module="review">
                                     <title>Pending Reviews</title>
-                                    <action>adminhtml/catalog_product_review/pending/</action>
+                                    <action>adminhtml/catalog_product_review/pending</action>
                                 </pending>
                                 <all translate="title" module="review">
                                     <title>All Reviews</title>
-                                    <action>adminhtml/catalog_product_review/</action>
+                                    <action>adminhtml/catalog_product_review/index</action>
```

Comment:
Why change this? It'll default to `index`.

Reply:
only to not have a ending slash